configure.ac

```diff
@@ -3930,17 +3930,17 @@ AS_CASE(["${YJIT_SUPPORT}"],
     ],
     [dev], [
 	rb_rust_target_subdir=debug
-	CARGO_BUILD_ARGS='--features disasm,runtime_checks'
+	CARGO_BUILD_ARGS='--features stats,disasm,runtime_checks'
 	AC_DEFINE(RUBY_DEBUG, 1)
     ],
     [dev_nodebug], [
 	rb_rust_target_subdir=dev_nodebug
-	CARGO_BUILD_ARGS='--profile dev_nodebug --features disasm'
+	CARGO_BUILD_ARGS='--profile dev_nodebug --features stats,disasm'
 	AC_DEFINE(YJIT_STATS, 1)
     ],
     [stats], [
 	rb_rust_target_subdir=stats
-	CARGO_BUILD_ARGS='--profile stats'
+	CARGO_BUILD_ARGS='--profile stats --features stats'
```

Comment:
This whole hunk should be removed since we've removed the stats feature. It's not listed in Cargo.toml anymore. There's some comments leftover referencing it I'll cleanup separtely.